financial-connections/src/main/java/com/stripe/android/financialconnections/repository/ConsumerSessionRepository.kt

```diff
@@ -44,7 +44,7 @@ internal class RealConsumerSessionRepository @Inject constructor(
         emailAddress = emailAddress,
         phoneNumber = getRedactedPhoneNumber(),
         clientSecret = clientSecret,
-        publishableKey = publishableKey,
+        publishableKey = null, // TODO(tillh-stripe): Coming in the follow-up
```

Comment:
Dow we reference `publishableKey` from `ConsumerSession` anywhere?

Reply:
Not yet. This will be used in the Instant Debits flow that we’re working on right now.